specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -97,6 +97,114 @@ export const attachmentsText = createDictionary({
     'uk-ua': 'Показати форму',
     'de-ch': 'Formular anzeigen',
   },
+  multipleFilesSelected: {
+    'en-us': 'Multiple files selected',
+  },
+  fileSize: {
+    'en-us': 'File Size',
+  },
+  deleted: {
+    'en-us': 'Deleted',
+  },
+  noFile: {
+    'en-us': 'No File',
+  },
+  incorrectFormatter: {
+    'en-us': 'Incorrectly Formatted',
+  },
+  alreadyUploaded: {
+    'en-us': 'Already Uploaded',
+  },
+  alreadyDeleted: {
+    'en-us': 'Already Deleted',
+  },
+  skipped: {
+    'en-us': 'Skipped',
+  },
+  cancelled: {
+    'en-us': 'Cancelled',
+  },
+  frontEndInterruption: {
+    'en-us': '{action:string} was in progress when interruption occured',
+  },
+
+  matchError: {
+    'en-us': 'Match Error',
+  },
+  errorReadingFile: {
+    'en-us': 'Error reading file',
+  },
+
+  unhandledFatalResourceError: {
+    'en-us': 'Unhandled fatal resource error:',
```

Comment:
very scarry error message!
hopefully no one will ever see this one

Reply:
Perhaps it is just my imagination, but I do recall an occurrence when something similar happened: when f.never() was called and the error message showed up in the console. 

Hopefully history doesn't repeat itself! :laughing:

Reply:
hopefully not! So much has to happen for it to occur. Some other transaction/user needs to change exactly the same attachment's attachment location and reorder it within the milliseconds. Also the attachment would have been made just now, so that's why this is very very unlikely, but handling it for completeness!

Reply:
lol, I had `defined()` function used in a lot of places where "typescript says this can be undefined, but I am smart, I know better than that - this can never happen"

@grantfitzsimmons kept on sending me screenshots from users with unhelpful errors messages "Value is not defined" (almost like "Null pointer exception" in sp6)

I ended up having to go though every place where we use `defined()`, and either handle the case gracefully without an error, or provide a propper helpful error message